### ISSUE TITLE
SuggestionTagsForUnnecessaryCode test is flaky

### ISSUE THREAD

**Issue by agocke** [dotnet/roslyn#6866]
See http://dotnet-ci.cloudapp.net/job/roslyn_prtest_win_dbg_unit32/1306/console

**Comment by davkean** [comment]
```
MESSAGE:
Assert.Equal() Failure\r\nExpected: 2\r\nActual: 0
+++++++++++++++++++
STACK TRACE:
at Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Squiggles.ErrorSquiggleProducerTests.VB$StateMachine_7_SuggestionTagsForUnnecessaryCode.MoveNext() in d:\j\workspace\roslyn_prtest_win_dbg_unit32\src\EditorFeatures\VisualBasicTest\Squiggles\ErrorSquiggleProducerTests.vb:line 96 --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c__DisplayClass2.<ThrowAsync>b__3(Object state)
```

**Comment by davkean** [comment]
@brettfo can you look at this soon? If you've got higher pri stuff, please load balance this.

**Comment by amcasey** [comment]
I just hit this.  How come it hasn't been skipped?

**Comment by agocke** [comment]
@amcasey It was. Force push ;)

**Comment by dpoeschl** [comment]
@amcasey @agocke Looking at this with @CyrusNajmabadi, it looks like an entire class of tests is problematic here, so just disabling one at a time won't fix it. We'll go ahead and disable all of the ErrorSquiggleProducerTests for C# & VB. @heejaechang -- Cyrus won't be available until next week, and there's a huge amount of async complexity in these tests. Do you have some bandwidth to take a look at how these tests wait on diagnostics/solutioncrawler. Ideally we'd split the solutioncrawler/asynchronous testing from the tagger testing to isolate which piece is flaky. Any thoughts?

**Comment by heejaechang** [comment]
I will take a look tomorrow.

**Comment by natidea** [comment]
@heejaechang Can we close this issue now? (Fixed by #7410)
